grafana/dashboards/cmode/volume.json

```diff
@@ -6451,7 +6451,7 @@
         "allValue": null,
         "current": {},
         "datasource": "${DS_PROMETHEUS}",
-        "definition": "label_values(volume_labels{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\"}, volume)",
+        "definition": "label_values(volume_labels{datacenter=~\"$Datacenter\",cluster=~\"$Cluster\",svm=~\"$SVM\",style!=\"flexgroup_constituent\"}, volume)",
```

Comment:
maybe style="flexvol" to avoid the negation

Reply:
The current Volume dashboard shows Flexvols and Flexgroup, we don't want to change the behaviour. The new Flexgroup dashboard would be shows granular data at flexgroup_constiituents.